torchrl/envs/batched_envs.py

```diff
@@ -586,6 +586,7 @@ def _reset(self, tensordict: TensorDictBase, **kwargs) -> TensorDictBase:
                         needs_resetting[i] += _reset[i].any()
                 else:
                     missing_reset = True
+                    # why do we break here?
```

Comment:
@matteobettini I don't undersetand why we break here.
We could have a reset that is missing somewhere (in which case we fall back on reset being an aggregate of the dones at this level) but other reset being present.
The risk is that someone partially populates the resets hoping that this will be passed to the sub-envs and then everything is silently reset.

Reply:
also the convention was that missing resets are considered all true, not an aggregate of dones